src/oauth-provider.ts

```diff
@@ -1391,6 +1421,9 @@ class OAuthProviderImpl {
     const now = Math.floor(Date.now() / 1000);
     const accessTokenExpiresAt = now + accessTokenTTL;
 
+    // Calculate refresh token expiration time if TTL is configured
+    const refreshTokenExpiresAt = refreshTokenTTL ? now + refreshTokenTTL : undefined;
```

Comment:
Technically this mishandles `refreshTokenTTL = 0`. Needs to explicitly check `refreshTokenTTL === undefined`.